aten/src/ATen/native/native_functions.yaml

```diff
@@ -4771,6 +4771,11 @@
 - func: sum.DimnameList_out(Tensor self, Dimname[1] dim, bool keepdim=False, *, ScalarType? dtype=None, Tensor(a!) out) -> Tensor(a!)
   device_check: NoCheck   # TensorIterator
 
+# TODO: this function will be replaced once nested expand semantics have been settled on
+- func: _nested_sum_backward(Tensor grad, Tensor self, int[1]? dim, bool keepdim=False) -> Tensor
+  dispatch:
+    NestedTensorCPU: NestedTensor_sum_dim_backward_CPU
```

Comment:
nit: the usual (from the rest of this file) naming convention for this would be `_nested_sum_backward_cpu`.
I know this is not the current convention for a few nested function but it would be good to align the two no?